PlaceRouteHierFlow/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM with_protobuf:latest as placeroute_image
+FROM darpaalign/with_protobuf:2019sept30 as placeroute_image
 
 RUN \
     git clone https://www.github.com/ALIGN-analoglayout/lpsolve.git  && \
     cp -r lpsolve /usr/local/lib/ && \
     apt-get update && \
-    apt-get install -yq libboost-all-dev git vim && \
+    apt-get install -yq vim && \
```

Comment:
Probably we don't want to be updating the apt-get system to install vim here. If we need it, we should also add it to the with_protobuf image. Probably, we don't need it and can get it on demand if we need to debug.